libr/util/print.c

```diff
@@ -1485,56 +1485,64 @@ R_API void r_print_bytes(RPrint *p, const ut8 *buf, int len, const char *fmt) {
 }
 
 R_API void r_print_raw(RPrint *p, ut64 addr, const ut8 *buf, int len, int offlines) {
-	if (offlines == 2) {
-		int i, j, cols = p->cols * 4;
-		char ch;
-		for (i = 0; i < len; i += cols) {
-			p->cb_printf ("0x%08"PFMT64x"  ", addr + i);
-			for (j = 0; j < cols; j++) {
-				if ((i + j) >= len) {
-					break;
+	switch (offlines) {
+		case 0:
+			p->write (buf, len);
+			break;
+		case 2:
+		{
+			int i, j, cols = p->cols * 4;
+			char ch;
+			for (i = 0; i < len; i += cols) {
+				p->cb_printf ("0x%08"PFMT64x"  ", addr + i);
+				for (j = 0; j < cols; j++) {
+					if ((i + j) >= len) {
+						break;
+					}
+					ch = buf[i + j];
+					if (p->cur_enabled) {
+						r_print_cursor (p, i + j, 1, 1);
+						p->cb_printf ("%c", IS_PRINTABLE (ch)? ch: ' ');
+						r_print_cursor (p, i + j, 1, 0);
+					} else {
+						p->cb_printf ("%c", IS_PRINTABLE (ch)? ch: ' ');
+					}
 				}
-				ch = buf[i + j];
-				if (p->cur_enabled) {
-					r_print_cursor (p, i + j, 1, 1);
-					p->cb_printf ("%c", IS_PRINTABLE (ch)? ch: ' ');
-					r_print_cursor (p, i + j, 1, 0);
+				p->cb_printf ("\n");
+			}
+			break;
+		}
+		default:
+		{
+			const ut8 *o, *q;
+			ut64 off;
+			int i, linenum_abs, mustbreak = 0, linenum = 1;
```

Comment:
mustbreak should be `bool`, not int

Reply:
I just made a copy paste. I am going to solve.